pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "pdl-api"
+version = "0.1.0"
+description = ""
+authors = ["Yong Wook Kim <ywkim@illinois.edu>"]
+readme = "README.md"
+packages = [{include = "pdl_api"}]
+
+[tool.poetry.dependencies]
+python = ">=3.7.1"
```

Comment:
I might suggest python="~3.9", no need to use 3.7. Also, instead of psycog2, it's better to use the binary, psycopg2-binary="~2.9"

Reply:
updated